polynote-frontend/polynote/scala.js

```diff
@@ -83,7 +83,6 @@ const tokenizer = {
 
       // delimiters and operators
       [/[{}()\[\]]/, '@brackets'],
-      [/[<>](?!@symbols)/, '@brackets'],
```

Comment:
I'm guessing this was in there to support XML, but then when I modified this grammar I probably removed the XML part and broke it.

Reply:
If anyone uses the scala XML support inside Polynote and complains about syntax highlighting I'll buy a hat and eat it.

Reply:
🎩